packages/prettier-plugin-java/src/printers/classes.js

```diff
@@ -50,23 +50,32 @@ class ClassesPrettierVisitor {
   }
 
   typeParameters(ctx) {
-    return "TBD";
+    const typeParameterList = this.visit(ctx.typeParameterList);
+
+    return rejectAndJoin("", ["<", typeParameterList, ">"]);
```

Comment:
maybe we should define a separate "compact" function?
- https://lodash.com/docs/4.17.11#compact

Reply:
maybe we should define our own wrappers on the DOC builders of prettier?
- always compact?
- always check for invalid arguments and throw errors?
  - I often had a bug poping up at  much later stage due to passing some null  value to the doc utils
     so it was hard to debug.